backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/types/CardActivationResultModel.kt

```diff
@@ -10,8 +10,16 @@ enum class ActivationState {
     failed,
 }
 
+@Suppress("ktlint:standard:enum-entry-name-case")
+enum class ActivationFailureReason {
```

Comment:
we already use ActivationState to pass a reason in some cases. 
for example, we have reasons "revoked" or "not_overwritten_existing" there.
so I would prefer not to add another enum with an additional set of reasons...

Reply:
So just adding (expired, not_found, wrong_secret) to ActivationState ?
Because it has already `did_not_overwrite_existing` it make sense to add the rest 🤔